api/XDS_PROTOCOL.md

```diff
@@ -345,6 +345,14 @@ resources to avoid resending them over the network.
 
 ![Incremental reconnect example](diagrams/incremental-reconnect.svg)
 
+#### Resource names
+Resources are identified by a resource name or an alias. Aliases of a resource, if present, can be identified by the alias field in the resource of a DeltaDiscoveryResponse. The resource name
```

Comment:
Nit: prefer to use backticks around proto messages like `DeltaDiscoveryResponse` for consistent formatting.

Reply:
Thanks updated.